tests/conftest.py

```diff
@@ -69,23 +104,45 @@ def create_mlwhdb_test_sessionfactory(config):
         conn.execute(text("SET foreign_key_checks=0;"))
         conn.commit()
     # Drop all tables and then create them to make it easier to test locally too.
-    Base.metadata.drop_all(bind=engine)
-    Base.metadata.create_all(bind=engine)
+    MlwhBase.metadata.drop_all(bind=engine)
+    MlwhBase.metadata.create_all(bind=engine)
+
+    return TestingSessionLocal
+
+
+@pytest.fixture(scope="function", name="qcdb_test_sessionfactory")
+def create_qcdb_test_sessionfactory(config):
+    engine = create_engine(qcdb_mysql_url(config), future=True)
+
+    TestingSessionLocal = sessionmaker(bind=engine)
+
+    # Drop all tables and then create then to make it easier to test locally too.
+    QcBase.metadata.drop_all(bind=engine)
+    QcBase.metadata.create_all(bind=engine)
 
     return TestingSessionLocal
 
 
 @pytest.fixture(scope="function", name="test_client")
-def create_test_client(config, mlwhdb_test_sessionfactory) -> TestClient:
+def create_test_client(
+    config, mlwhdb_test_sessionfactory, qcdb_test_sessionfactory
+) -> TestClient:
     def override_get_mlwh_db():
         try:
             db: Session = mlwhdb_test_sessionfactory()
             yield db
         finally:
             db.close()
 
-    app.dependency_overrides[get_mlwh_db] = override_get_mlwh_db
+    def override_get_qc_db():
+        try:
+            db: Session = qcdb_test_sessionfactory()
+            yield db
+        finally:
+            db.close()
 
+    app.dependency_overrides[get_mlwh_db] = override_get_mlwh_db
```

Comment:
I look forward to some functionality that uses both of these at once.

Reply:
That should come in the next PR, with the filtered inbox view tests 🙂